### PR TITLE
Add change user feature for debugging

### LINKED ISSUE #126: Add feature to test data for different hiring leads

### PR DESCRIPTION
**Pull request by andesol**
## Done
Add dropdown to test data for different hiring leads 
## QA
- Start database and run the project
```
docker-compose up -d
dotrun
```
- Go to http://localhost:8501/stages/application-review
- Check data changes when differents hiring leads are selected in the dropdown

 ## Issue
Fixes #126

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -37,8 +37,36 @@ def index():
...
+
+# Debugging endpoint
+# To be removed
+@app.route("/change-user", methods=["POST"])
+@login_required
+def change_user():
+    """Change user for debugging purposes."""
+
+    if DEBUG:
+
```
Comment (tbille) [inline webapp/app.py:55]: No need for extra space

File: webapp/app.py [hunk 2]
```diff
@@ -37,8 +37,36 @@ def index():
...
+@app.route("/user.json")
 @login_required
 def api_user():
     """Get user information"""
-    return flask.jsonify(flask.session["openid"]["fullname"])
+    return flask.jsonify(flask.session["openid"])
+
+
+# Debugging endpoint
+# To be removed
```
Comment (tbille) [inline webapp/app.py:48]: No need for those comments.
I think we should keep this enpoint, but it should only be available if I am in debug mode. We opened an issue to track this: https://github.com/canonical-web-and-design/hiring.canonical.com/issues/127

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -127,7 +127,6 @@ jobs:
       - name: Run JS tests with coverage
         run: |
           yarn test-js
-          bash <(curl -s https://codecov.io/bash) -cF typescript
```
Comment (tbille) [inline .github/workflows/pr.yaml:130]: Are we removing this section for a reason?

Comment (andesol) [reply .github/workflows/pr.yaml]: It seemed to be causing the `test-js` action to fail randomly. But the issue seems to be somewhere else, so just brought it back again.

**Comment by andesol** [comment]
@tbille Changes done!

File: webapp/app.py (hunk 1 above)
Comment (andesol) [reply webapp/app.py]: :+1:

File: webapp/app.py (hunk 2 above)
Comment (andesol) [reply webapp/app.py]: :+1:

**Review by tbille — APPROVED** [review]
Let's :ship:  it!

**Pull request merged**